composer.json

```diff
@@ -33,6 +33,7 @@
         "illuminate/pipeline": "^7.0",
         "illuminate/queue": "^7.0",
         "illuminate/support": "^7.0",
+        "illuminate/testing": "^7.0",
```

Comment:
Please move this to `require-dev` and make it a "suggested" dependency.

Reply:
> Please move this to require-dev and make it a "suggested" dependency.

If you want to make a HTTP tests on your Lumen app, then you need to manually add the dependency, which is different to Laravel where this would be available by default.

Reply:
Yes, that is correct. Lumen is intentionally light out of the box. Lots of stuff doesn't work unless you opt in to it.

Reply:
We should include this dependency in the example `laravel/lumen` app's require-dev too.

Reply:
And in the upgrading guide.